src/20_Components/amp-fx-flying-carpet.html

```diff
@@ -100,14 +100,13 @@
 
      The text that follows is to ensure that the `amp-fx-flying-carpet` is placed before the last viewport.
   -->
-  <p>
+  <p class="p1">
```

Comment:
I added that rule at the component sample level for amp-bind, should I move it to example.css?